storage/database/shardedPersister_test.go

```diff
@@ -351,6 +354,7 @@ func getKey(
 		defer func() {
 			db.Close()
```

Comment:
also a stop timer before close and a start timer after close in the deferred func